tests/test_examples.py

```diff
@@ -46,6 +47,9 @@ def test_5():
 
     test_resources_in_dataframe(dirname, fhir_api_url, path_rules)
 
+def test_6():
+    create_resource_test("tests/6")
```

Comment:
What does this feature test on FHIR2Dataset? I get the impression that it only creates resources on the API, is this the case?

Reply:
Yes you are right, I used it to test something else. I meant to delete it but I forgot that sorry

Reply:
No problem, I'm not suggesting we remove these unwanted changes at this time.